api/context/src/main/java/io/opentelemetry/context/StrictContextStorage.java

```diff
@@ -92,6 +92,17 @@ public Scope attach(Context context) {
     CallerStackTrace caller = new CallerStackTrace(context);
     StackTraceElement[] stackTrace = caller.getStackTrace();
 
+    // Detect invalid use from kotlin.
+    for (StackTraceElement element : stackTrace) {
+      if (element.getClassName().equals("kotlin.coroutines.jvm.internal.BaseContinuationImpl")
```

Comment:
urgh. Any thoughts on how to keep this up-to-date if kotlin renames or changes their continuation implementation? Do we know what versions of kotlin this will work for, at this point?

Reply:
as you say...best effort is better than nothing! Just wondering if we can provide some documentation about what is supported by this.

Reply:
I added some doc to `Context` about the use of internal details, not sure how better to phrase it.